spec/spec_helper.rb

```diff
@@ -12,7 +12,7 @@
 unless ENV['COSMOS_NO_SIMPLECOV']
   require 'simplecov'
   require 'coveralls'
-  Coveralls.wear!
+  Coveralls.wear_merged!
```

Comment:
I don't think this matters on the Travis build side but thought we might need it when running locally with AHK.